ssl/ssl_lib.c

```diff
@@ -5773,8 +5773,7 @@ static int ct_move_scts(STACK_OF(SCT) **dst, STACK_OF(SCT) *src,
         }
     }
 
-    while (sk_SCT_num(src) > 0) {
-        sct = sk_SCT_pop(src);
```

Comment:
I disagree with reverting such cleanups of sloppy stack API uses that were done in  https://github.com/openssl/openssl/commit/30eba7f35983a917f1007bce45040c0af3442e42.

Going back to, e.g., `while ((sct = sk_SCT_pop(src)) != NULL)` does not contribute to fixing the small regression reported in #19389, and in IMO would be even counter-productive.

Reply:
And I disagree with this as if the sk_pop on empty stack is legitimate call, and I insist that it is, there is nothing wrong with it to be used across the code.

Reply:
At least the following _is_ wrong/problematic with `while ((sct = sk_SCT_pop(src)) != NULL)`:
it does not distinguish between NULL entries on the stack, the empty stack, and a NULL stack pointer.

Reply:
I'd prefer these to remain, however I can live with removing them for the moment.  They can be reintroduced separately.